### PR TITLE
build: WIP: upgrading to material for mkdocs 5

### PR DESCRIPTION
**Pull request by Philip-Larsen-Donnelly**
Needs a fix for the version and language selectors on the left TOC, but otherwise looks OK.

### CODE REVIEW

File: src/commonmark/en/resources/css/mkdocs_dhis2.css [hunk 1]
```diff
@@ -280,12 +280,17 @@ article {
   color: var(--grey600);
 }
 
+.md-nav__link.md-nav__link--active {
+  color: var(--blue700);
```
Comment (cooper-joe) [inline src/commonmark/en/resources/css/mkdocs_dhis2.css:284]: I'd also add
`background: var(--blue100)` and `border-radius: 3px;` here to make it even clearer that it is selected.

Comment (cooper-joe) [reply src/commonmark/en/resources/css/mkdocs_dhis2.css:284]: This also looks better if you add `padding: 2px;` to `.md-nav__link` class as well.

**Comment by cooper-joe** [comment]
I've added some comments to make the selected section clearer.

I can't see a css fix for the scrolling-out-of-view selectors. They are inside the `.md-nav` which is a scrolling container.

**Comment by Philip-Larsen-Donnelly** [comment]
Thanks @cooper-joe .

I can move the selectors. I tried putting them after the Label (e.g. "Implementer") and before the first document. Then they were visible, but it looked wrong to me.
I may need a js wizard to tell me what causes it to jump to the label and how we can make it jump to the selectors instead. My only thought was to hack it with a hidden label, then the selectors, then a second, visible, label. :/

**Pull request merged**